index.bs

```diff
@@ -2657,10 +2919,10 @@ partial interface MLGraphBuilder {
 <div>
     **Arguments:**
         - *input*: an {{MLOperand}}. The input N-D tensor from which the values are gathered.
-        - *indices*: an {{MLOperand}}. The indices N-D tensor of the input values to gather. The values must be in the range [0, N] where N is the value of the input shape indexed by *options.axis*.
+        - *indices*: an {{MLOperand}}. The indices N-D tensor of the input values to gather. The values must be of type "uint32" in the range [0, N-1] where N is the size of the input dimension indexed by *options.axis*.
```

Comment:
> The values must be of type "uint32"

We will need to handle int64 too because existing models have int64 `gather` indices, and `argMin`/`argMax`'s own output data type is int64 (meaning you couldn't connect `argMin`'s output to `gather` without a cast otherwise). I'm content to resolve this comment though and continue with the CR given we have this relevant issue https://github.com/webmachinelearning/webnn/issues/484.